spec/models/printer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Printer, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I think you could add in just a couple of tests in here to make sure Printers aren't added without a name or label type e.g.

```ruby
it 'is not valid without a name' do
  expect(build(:printer, name: nil)).to_not be_valid
end
```